### PR TITLE
Renovation: file-name.ext -> file_name.ext

### PR DESCRIPTION
**Pull request by churkin**

### CODE REVIEW

File: testing/jest/tests/button.tests.tsx [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
   EVENT,
-} from './utils/events-mock';
-import Button, { ButtonProps, defaultOptionRules, viewFunction } from '../../js/renovation/button';
-import Widget from '../../js/renovation/widget';
-import Icon from '../../js/renovation/icon';
-import InkRipple from '../../js/renovation/ink-ripple';
+} from '../utils/events_mock';
+import Button, { ButtonProps, defaultOptionRules, viewFunction } from '../../../js/renovation/button';
+import Widget from '../../../js/renovation/widget';
+import Icon from '../../../js/renovation/icon';
```
Comment (MaximKudriavtsev) [inline testing/jest/tests/button.tests.tsx:15]: Should Icon be at `shared` folder?

Comment (churkin) [reply testing/jest/tests/button.tests.tsx:15]: Yes, in the next PR

Comment (churkin) [reply testing/jest/tests/button.tests.tsx:15]: Yep, in the next PR

Comment (MaximKudriavtsev) [reply testing/jest/tests/button.tests.tsx:15]: `Yep`... are you American? 😂

**Review by MaximKudriavtsev — APPROVED** [review]

**Pull request merged**